### PR TITLE
Docs/install

### PR DESCRIPTION
**Pull request by kilosonc**

### CODE REVIEW

File: docs/tutorials/how-to-install/local-installation.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+:::
+
+</TabItem>
+</Tabs>
+
+### Install `Horizon` by helm
+
+Install `Horizon` by helm, you can find the latest version of `Horizon` in [Horizon Chart Repo](https://github.com/horizoncd/helm-charts/releases).
+
+The default installation is not the same as the minimal installation by script, it will install install `gitlab` by default.
```
Comment (iamyeka) [inline docs/tutorials/how-to-install/local-installation.md:274]: Two install word

File: docs/tutorials/how-to-install/local-installation.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+```
+
+:::
+
+```bash
+helm repo add horizoncd https://horizoncd.github.io/helm-charts
+helm install horizon horizoncd/horizon -n horizoncd --version 2.1.7 --create-namespace
+```
+
+Keep watching the service status of `Horizon`. If everything goes well, you can see the following output:
```
Comment (iamyeka) [inline docs/tutorials/how-to-install/local-installation.md:310]: Do we need to point out that the output below is based on full installation?

File: docs/tutorials/how-to-install/local-installation.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+# tekton-triggers-webhook-559b5b98cc-c7tmj             1/1     Running     0          40m
+```
+
+### Check the installation result
+
+If everything goes well, you can enter Horizon by visiting the url: http://horizon.h8r.site, and you will see the page like below:
+
+<img src="/image/home.png" style={{boxShadow: "10px 5px 5px grey", radius:"10px", borderRadius: "10px"}} />
+
+Next, please go to [how to deploy your fist workload](/docs/tutorials/how-to-deploy-your-first-workload) to experience this amazing system more closely.
```
Comment (iamyeka) [inline docs/tutorials/how-to-install/local-installation.md:351]: Do we need to tell users that they don't need to follow `how to deploy your fist workload` guide if they install horizon by installation script?

File: docs/tutorials/how-to-install/local-installation.md (hunk 1 above)
Comment (kilosonc) [reply docs/tutorials/how-to-install/local-installation.md:274]: fixed

File: docs/tutorials/how-to-install/local-installation.md (hunk 3 above)
Comment (kilosonc) [reply docs/tutorials/how-to-install/local-installation.md:351]: fixed

File: docs/tutorials/how-to-install/local-installation.md (hunk 2 above)
Comment (kilosonc) [reply docs/tutorials/how-to-install/local-installation.md:310]: fixed

**Review by iamyeka — APPROVED** [review]

**Review by xuzhu-591 — APPROVED** [review]

**Review by xuzhu-591 — APPROVED** [review]

**Pull request merged**